src/generateNumberCards.js

```diff
@@ -0,0 +1,25 @@
+function generateRandomInt(max, min) {
+    return Math.floor(Math.random() * (max - min + 1) + min)
+}
+
+function generate3To5NumberCards(){
+let randomNumbersHolder = {}
+    for (let i = 0; i < generateRandomInt(5,3); i++) {
+        let randomNumber = generateRandomInt(100, 0)
+        while (Object.values(randomNumbersHolder).includes(randomNumber)) {
+            randomNumber = generateRandomInt(100, 0)
+        } 
+        randomNumbersHolder[i] = randomNumber
+    }
+
+Object.values(randomNumbersHolder).forEach(number => {
```

Comment:
again, if this was an array to start with, you wouldn't have to get the values out, you could just forEach it

Reply:
also, this loop is unnecessary, your already looping the right number of times in for `for` loop above, why not just assign to the DOM at the same time?

Reply:
agreed, I have now assigned the random number to the DOM within the first loop